### PR TITLE
docs: large update why quarto

### PR DESCRIPTION
**Pull request by K-Beicher**
## Description

- This PR moves the existing text for why quarto to the new template, there is a separate issue (#40) created to deal with all the missing information. 
## Related Issues

Closes This is one of many PRs based on a single issue

## Reviewer Focus

This PR requires an in-depth review.

There is a substantial amount of information missing for this document, as it stands it is more like a stump than an actual why document.

## Checklist

For general documentation:

- [x ] Spell-check
    - [ ] US
    - [ ] UK
- [ x] Did the page(s) preview correctly on your machine without breaking
- [ x] New category words (keywords) (if any) added to the code snippet file

### CODE REVIEW

**Review by signekb — DISMISSED** [review]
I'll approve, since it now fits the template (even though some info is still missing) :)

**Comment by lwjohnst86** [comment]
@seedcase-project/developers I've updated this post to include comparisons with other tools.

File: why-quarto.qmd [hunk 1]
```diff
@@ -1,51 +1,202 @@
...
+:::
 
-For these reasons, we write our documentation that includes the website
-with Quarto.
+We have decided to go with Quarto as this seems to be the most
+accessible, as well as being a tool that there is already support for
+internally in the team. It has the most functionality when it comes to
+features, theming, and customization, while still being very easy to use
+if you have very basic knowledge of Pandoc Markdown, building websites,
+and file paths.
```
Comment (signekb) [inline why-quarto.qmd:202]: Is the template header "Consequences" optional or is it missing here?

**Review by signekb — COMMENTED** [review]
Very nice 👍 I have added some suggestions and questions

**Comment by K-Beicher** [comment]
@lwjohnst86 If I change the owner of this PR to you, will you be able to merge the changes that Signe has suggested? As far as I can see, they are all comments to the text you added.

File: why-quarto.qmd (hunk 1 above)
Comment (lwjohnst86) [reply why-quarto.qmd:201]: I think its optional if there aren't any major consequences.

**Comment by lwjohnst86** [comment]
@signekb made some changes based on your suggestions 🎉

**Comment by lwjohnst86** [comment]
@K-Beicher since I am the owner of the Seedcase Project GitHub repo, I can merge and handle changes just fine ☺️

File: why-quarto.qmd [hunk 2]
```diff
@@ -22,11 +20,12 @@ State the context and some background on the issue, then write a
...
 :::
 
-A software project like Seedcase needs an easy way to communicate with
-potential users that might not be familiar with using Github. The
-easiest way to do this is by having a website. In order to minimize the
-work the team needs to do, we would like to use the files we maintain in
-GitHub as the basis for a public website. The question then becomes:
+A software project like Seedcase needs an easy way to communicate and
+share knowledge for those internal and external to Seedcase (including
+users), the most common way being to put these content (like
```
Comment (signekb) [inline why-quarto.qmd:25]:
```suggestion
users and contributors), the most common way being to put these content (like
```
maybe? :)

**Review by signekb — COMMENTED** [review]
Very nice! 🎉 Just one suggestion this time.

**Pull request merged**